pkg/crc/machine/start.go

```diff
@@ -449,6 +440,11 @@ func (client *client) Start(ctx context.Context, startConfig types.StartConfig)
 
 	waitForProxyPropagation(ctx, ocConfig, proxyConfig)
 
+	clusterConfig, err := getClusterConfig(crcBundleMetadata)
+	if err != nil {
+		return nil, errors.Wrap(err, "Cannot create cluster configuration")
```

Comment:
Maybe change `create` to something more accurate to what it's doing? `read` or `get`?

Reply:
Updated.